presto-product-tests/conf/tempto/tempto-configuration-for-docker-kerberos-simba.yaml

```diff
@@ -0,0 +1,47 @@
+hdfs:
```

Comment:
This file is a copy of tempto-configuration-for-docker-kerberos.yaml.  The only parts that are different are the JDBC class and JDBC URL.  If there is a way to override those two lines without duplicating the file, I would much prefer to do that.